SwiftGen.xcodeproj/project.pbxproj

```diff
@@ -548,12 +548,14 @@
 			isa = PBXNativeTarget;
 			buildConfigurationList = 09A87B451BCC9B8000D9B9F5 /* Build configuration list for PBXNativeTarget "swiftgen" */;
 			buildPhases = (
+				54570B6ACBB962B5ED01A405 /* [CP] Check Pods Manifest.lock */,
 				BEC9F3A158B764C68AF03EF9 /* [CP] Check Pods Manifest.lock */,
```

Comment:
Seems strange that the Build Phase got duplicated here. We gotta double-check that.